src/OrchardCore.Modules/OrchardCore.Https/Startup.cs

```diff
@@ -34,6 +46,13 @@ public override void ConfigureServices(IServiceCollection services)
             services.AddSingleton(new RewriteOptions());
             
             services.TryAddEnumerable(ServiceDescriptor.Transient<IConfigureOptions<RewriteOptions>, RewriteOptionsHttpsConfiguration>());
+
+            services.AddHsts(options =>
+            {
+                options.Preload = true;
+                options.IncludeSubDomains = true;                
+            });
```

Comment:
We should set the MaxAge explicitly since it defaults to only 30 days. I've seen 365 days used commonly in other projects.